main.py

```diff
@@ -33,4 +33,14 @@ def get_patients(skip: int = 0, limit: int = 100, db: Session = Depends(get_db))
 @app.get("/physicians/", response_model=List[response_models.Physician])
 def get_physicians(skip: int = 0, limit: int = 100, db: Session = Depends(get_db)):
     physicians = physician_crud.get_physicians(db, skip=skip, limit=limit)
-    return physicians
\ No newline at end of file
+    return physicians
+
```

Comment:
same comment as above. All non-appointment-related code should be handled by other team